Infrastructure/Constants.cs

```diff
@@ -0,0 +1,13 @@
+namespace WebApp_OpenIDConnect_DotNet.Infrastructure
+{
+    public static class Constants
+    {
+        public const string AdditionalClaims = "claims";
+        public const string OpenIdResponseType = "id_token code";
```

Comment:
We might be able to use `OpenIdConnectResponseType.CodeIdToken`
(I need to check if the order matters)